### DISCUSSION TITLE
V4升级到V5遇到的问题

### DISCUSSION THREAD

**Discussion by lkzwc** [ant-design/ant-design#41470] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84896877/227833583-333610e7-6793-4839-b008-3672a2a3ee6c.png)
V4中修改的样式 在less中使用的都是变量，V5版本不用引入了，那么之前的这些变量要一一用具体的替换吗？大家有没有什么经验

**Comment by yoyo837** [comment]
这里 https://github.com/ant-design/ant-design/issues/38975

**Comment by arvinxx** [comment, marked as answer]
如果你想整体转 cssinjs的话，可以等一等，这周就会有 codemod 了。

**Reply by lkzwc** [reply]
好的

**Reply by lkzwc** [reply]
> 如果你想整体转 cssinjs的话，可以等一等，这周就会有 codemod 了。
有没有结果呀

**Reply by arvinxx** [reply]
核心功能已经ready ，在这里 https://github.com/chenshuai2144/less2cssinjs


import { transformCssAndTsx } from '@chenshuai2144/less2cssinjs/dist/esm/findAllLess';


transformCssAndTsx('xxx');

Xxx是路径，会将less转成cssinjs，然后写对应的文件

还没写成命令行脚本，着急用的话可以自己先写一个，应该还简单的。
